src/time_integration/methods_SSP.jl

```diff
@@ -162,6 +147,20 @@ function solve(ode::ODEProblem, alg = SimpleSSPRK33()::SimpleAlgorithmSSP;
         init_callback(stage_callback, integrator.p)
     end
 
+    return integrator
+end
+
+"""
+    solve(ode, alg; dt, callbacks, kwargs...)
+
+The following structures and methods provide the infrastructure for SSP Runge-Kutta methods
+of type `SimpleAlgorithmSSP`.
+"""
+function solve(ode::ODEProblem, alg::SimpleAlgorithmSSP;
+               dt, callback::Union{CallbackSet, Nothing} = nothing, kwargs...)
```

Comment:
```suggestion
function solve(ode::ODEProblem, alg::SimpleAlgorithmSSP;
               dt, callback = nothing, kwargs...)
```
For the other time integration methods we only have `callback = nothing` and no `::Union{CallbackSet, Nothing}` anymore. Do we want to have this here as well?

Reply:
Also, isn't the function `solve()` now identical for all 4 custom time integration methods? Maybe we can also have one version for all in `time_integration.jl`.